cockatrice/src/lineeditcompleter.cpp

```diff
@@ -1,133 +1,205 @@
 #include <QStringListModel>
 #include <QLineEdit>
 #include <QCompleter>
-#include <QWidget>
-#include <QTextCursor>
 #include <QAbstractItemView>
 #include <QScrollBar>
-#include <QStringListModel>
 #include <QKeyEvent>
-#include <QFocusEvent>
```

Comment:
Removing includes either will cause no issues or compiling will fail, so
that's not it.

On Wed, Jul 6, 2016 at 12:40 PM Zach <notifications@github.com> wrote:

> In cockatrice/src/lineeditcompleter.cpp
> <https://github.com/Cockatrice/Cockatrice/pull/2053#discussion_r69766298>:
>
> >  #include <QKeyEvent>
> > -#include <QFocusEvent>
>
> I have a feeling removing all these includes is the reason windows is
> crashing
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Cockatrice/Cockatrice/pull/2053/files/b4dec04bb64b1f42b7d419cb72d7fe694e361767#r69766298>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAA5NMnps5ytIBUKsXx5yiZMY1uuBBxCks5qS9pkgaJpZM4JFje->
> .
>